css/card.css

```diff
@@ -570,7 +570,6 @@ section#gacha-controls fieldset:nth-child(2) > input:first-child {
   margin: 20px;
   overflow: hidden;
   z-index: 10;
-  touch-action: none;
```

Comment:
@kirinarimasu I am not sure if removing this affects anything. This was causing the user to be unable to scroll on mobile. The card was eating the scroll event

Reply:
I think this is okay to remove, thanks! It doesn't make sense that it's there anymore, nice catch!